code/modules/projectiles/guns/ballistic/magweapon.dm

```diff
@@ -77,15 +77,17 @@
 
 /obj/item/gun/ballistic/automatic/magrifle/pistol
 	name = "magpistol"
-	desc = "A handgun utilizing maglev technologies to propel a ferromagnetic slug to extreme velocities."
+	desc = "A handgun utilizing maglev technologies to propel a ferromagnetic slug to extreme velocities. Packs less of a punch."
 	icon_state = "magpistol"
 	w_class = WEIGHT_CLASS_NORMAL
 	slot_flags = ITEM_SLOT_BELT
 	fire_sound = 'sound/weapons/magpistol.ogg'
 	mag_type = /obj/item/ammo_box/magazine/mmag/small
 	fire_delay = 2
 	inaccuracy_modifier = 0.25
-	cell_type = /obj/item/stock_parts/cell/magnetic/pistol
+	cell_type = /obj/item/stock_parts/cell/magnetic/pistol //We also hold less ammo
```

Comment:
We now use the same power as are bigger verson, but hold less cell charge for ammo shooting. To keep balanced